dma4500mSup/dma4500m.proto

```diff
@@ -5,8 +5,8 @@
 # European Spallation Source ERIC
 # March 2018
 
-replytimeout = 2000;
-readtimeout  = 2000;
+replytimeout = 5000;
+readtimeout  = 5000;
 terminator = CR;
 
 # Make sure stream buffer is empty
```

Comment:
In `set_temperature` can you redirect the returned values to a PV. If automeasure is on setting the temperature silently fails, so I think this feedback would be useful to a user

Reply:
I can, but the `TEMPERATURE:SP` PV gets disabled during a measurement so `set_temperature` wouldn't be executed at all if a measurement is running. In other words, the PV holding the returned value would only ever be "accepted", as the command isn't executed when it would return "not allowed during measurement"